src/components/student/ScheduleView.js

```diff
@@ -9,14 +14,130 @@ import React, {useState} from 'react';
 
 const ScheduleView = (props) => {
-    
-   
-    return(
-        < > 
-            <h3>Not implemented</h3>
-        </ >
-    );
-
-}
+
+    const headers = ['SecNo', 'CourseId', 'SecId',  'Year', 'Semester', 'Building', 'Room', 'Times', '', ''];
+
+    const [search, setSearch] = useState({
+        scheduleId: '',
+        year: '',
+        semester: ''
+    });
+    const [message, setMessage] = useState('');
+    const [schedules, setSchedules] = useState([]);
+
+    const fetchSections = async () => {
+            if (search.courseId==='' || search.year==='' || search.semester==='' ) {
+                setMessage("Enter search parameters");
+            } else {
+              try {
+                const response = await fetch(`${SERVER_URL}/courses/${search.courseId}/sections?year=${search.year}&semester=${search.semester}`);
+                if (response.ok) {
+                  const data = await response.json();
+                  setSchedules(data);
+                } else {
+                  const rc = await response.json();
+                  setMessage(rc.message);
+                }
+              } catch(err) {
+                setMessage("network error: "+err);
+              }
+            }
+        }
+
+        const deleteSchedule = async (secNo) => {
+          try {
+            const response = await fetch (`${SERVER_URL}/sections/${secNo}`,
+            {
+              method: 'DELETE',
+              headers: {
+                'Content-Type': 'application/json',
+              },
+            });
+            if (response.ok) {
+              setMessage("Schedule deleted");
+              fetchSections();
+            } else {
+              const rc = await response.json();
+              setMessage(rc.message);
+            }
+          } catch (err) {
+            setMessage("network error: "+err);
+          }
+        }
+
+
+        const editChange = (event) => {
+            setSearch({...search,  [event.target.name]:event.target.value});
+        }
+
+        const onDelete = (SecNo) => {
+          confirmAlert({
+              title: 'Confirm to delete',
+              message: 'Do you really want to delete?',
+              buttons: [
+                {
+                  label: 'Yes',
+                  onClick: () => deleteSchedule(SecNo)
+                },
+                {
+                  label: 'No',
+                }
+              ]
+            });
+        }
+
+
+      return(
+              <div>
+                  <h3>Schedules!!!</h3>
+
+                  <h4>{message}</h4>
+                  <h4>Enter course prefix, year, semester.  Example  cst 2024 Spring</h4>
+                  <table className="Center">
+                      <tbody>
+                      <tr>
+                          <td>Course Prefix:</td>
+                          <td><input type="text" id="scourseId" name="courseId" value={search.courseId} onChange={editChange} /></td>
```

Comment:
We can delete the courseId input. See comments above.

Reply:
Removed the Course Prefix section successfully.